stable/application-mesh/tests/golden/components/envoy-filter-max-body-size.values.yaml

```diff
@@ -0,0 +1,14 @@
+global:
+  application:
+    product: leads-broker
+    environment: test
+    version: "0.0.0"
+
+destinations:
+  - component: webserver
+
+ingress:
+  - host: leads-broker-st1.matic.com
+    gateway: external-matic-com
+    destination: webserver
+    max_request_bytes: "67108864"
```

Comment:
I would recommend adding one level of nesting here and making a key a bit more clear:
```
- host: leads-broker.matic.com
   settings:
     max_request_size_bytes: 1234
```